lib/stringify.js

```diff
@@ -7,6 +7,7 @@ var arrayPrefixGenerators = {
     brackets: function brackets(prefix) { // eslint-disable-line func-name-matching
         return prefix + '[]';
     },
+    comma: 'comma',
```

Comment:
why is this one not a function? that breaks with the established convention here.

Reply:
Because 'comma' is differ to others in implementation, but would be more make sense to set in `arrayFormat`.

For example, what `brackets` or others do is try to iterate over each item in an array, with a well-documented key part in result, such as
``` 
[a,b,c].map(item => `${key(item)}=${item}`
```
, while `comma` is more like a joint stringify on value part
```
`${key(items)}=${[a,b,c].join(',')}`
```
in turn , `arrayPrefixGenerators['comma']` has no meaning as a function.